File: internal/handlers/logout.go
package handlers

import (
	"github.com/gin-gonic/gin"
	"github.com/rs/zerolog/log"
)

func (h *Handlers) LogoutHandler(c *gin.Context) {
	log.Debug().Msg("Cleaning up redirect cookie")
	h.Auth.DeleteSessionCookie(c)
	c.JSON(200, gin.H{
		"status":  200,
		"message": "Logged out",
	})
}

File: internal/handlers/Logout.cs
using Tinylinks.Auth;

namespace Tinylinks.Handlers
{
    public partial class Handlers
    {
        public void LogoutHandler(Auth.Context ctx)
        {
            Auth.DeleteSessionCookie(ctx);
            ctx.JSON(200, new { status = 200, message = "Logged out" });
        }
    }
}
